doc/translations.md

```diff
@@ -3,78 +3,52 @@ Friendica translations
+Please register a free Transifex account and ask over at [the Transifex Friendica page](https://www.transifex.com/Friendica/friendica/dashboard/) to join the translation team for your favorite language.
 
-If you want to get your work into the source tree yourself, feel free to do so and contact us with and questions that arise.
-The process is simple and friendica ships with all the tools necessary.
+As a rule of thumb, we add support for a language in Friendica when at least 50% of the strings have been translated to avoid a scattered experience.
+For addons, we add support for a language when all the strings for this addon have been translated and only if we already support the language in Friendica.
 
-The location of the translated files in the source tree is
-    /view/lang/LNG-CODE/
-where LNG-CODE is the language code used, e.g. de for German or fr for French.
-The translated strings come as a "message.po" file from transifex which needs to be translated into the PHP file friendica uses.
-To do so, place the file in the directory mentioned above and use the "po2php" command from the Friendica Console.
+## Add new translation strings
 
-Assuming you want to convert the German localization which is placed in view/lang/de/message.po you would do the following.
+### Core
 
-    1. Navigate at the command prompt to the base directory of your
-       friendica installation
+Once you have added new translation strings in your code changes, please run `bin/run_xgettext.sh` from the base Friendica directory and commit the updated `view/lang/C/messages.po` to your branch.
 
-    2. Execute the po2php command, which will place the translation
-       in the strings.php file that is used by friendica.
+### Addon
 
-       $> php bin/console.php po2php view/lang/de/messages.po
+If you have the `friendica-addons` repository in the `addon` directory of your Friendica cloned repository, just run `bin/run_xgettext.sh -a <addon_name>` from the base Friendica directory.
 
-       The output of the script will be placed at view/lang/de/strings.php where
-       friendica is expecting it, so you can test your translation immediately.
-       *Please note that the console tool has to be called from the base directory of your Friendica installation.*
+Otherwise:
 
-    3. Visit your friendica page to check if it still works in the language you
-       just translated. If not try to find the error, most likely PHP will give
-       you a hint in the log/warnings.about the error.
+	cd /path/to/friendica-addons/<addon_name>
+	/path/to/friendica/bin/run_xgettext.sh -s
 
-       For debugging you can also try to "run" the file with PHP. This should
-       not give any output if the file is ok but might give a hint for
-       searching the bug in the file.
+In either case, you need to commit the updated `<addon_name>/lang/C/messages.po`.
 
-       $> php view/lang/de/strings.php
+## Update translations from Transifex
```

Comment:
This step should only be needed if you want to update the translation in the projects repository. In this case I think it should be mentioned that both files, `messages.po` and `strings.php` should be committed to the repository.